packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -62,18 +65,11 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
     fontFamily: typography.fontByWeight.demiBold,
     fontSize: typography.fontSize.medium,
     color: palette.neutral,
-    lineHeight: spacing(6)
-  },
-  checkIcon: {
-    width: spacing(6),
-    height: spacing(6)
+    lineHeight: typography.fontSize.medium * 1.3
```

Comment:
oof, would be nice to get the typography stuff into mobile as well? Shouldn't need to set line heights manually...

Reply:
totally agree but out of scope for now!